README.md

```diff
@@ -7,3 +7,93 @@ Tool for converting specialized annotated Verilog models into XML needed for
 
```

Comment:
We should make sure the contents of this `README.md` appear in the readthedocs output without having to duplicate it all.